librealsense: 2.25.0 -> 2.29.0

r-ryantm opened the pull request:
Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools. This update was made based on information from https://repology.org/metapackage/librealsense/versions.



meta.description for librealsense is: '"A cross-platform library for Intel® RealSense™ depth cameras (D400 series and the SR300)"'.


meta.homepage for librealsense is: '"https://github.com/IntelRealSense/librealsense"

[Release on GitHub](https://github.com/IntelRealSense/librealsense/releases/tag/v2.28.0)


[Compare changes on GitHub](https://github.com/IntelRealSense/librealsense/compare/v2.25.0...v2.28.0)

<details>
<summary>
Checks done (click to expand)
</summary>

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 2.28.0 with grep in /nix/store/wcqkph9wihh4mg0hkg1vq846iv3rwc1m-librealsense-2.28.0
- directory tree listing: https://gist.github.com/3f24c75ec4db7ff7cd0ec8e9f3810387

- du listing: https://gist.github.com/6e755e75f9548c0bd8b44bf8d4fd7cbd


</details>
<details>
<summary>
Rebuild report (if merged into master) (click to expand)
</summary>

3 total rebuild path(s)

1 package rebuild(s)

1 x86_64-linux rebuild(s)
1 i686-linux rebuild(s)
1 x86_64-darwin rebuild(s)
0 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
librealsense

</details>

<details>
<summary>
Instructions to test this update (click to expand)
</summary>

Either download from Cachix:
```
nix-store -r /nix/store/wcqkph9wihh4mg0hkg1vq846iv3rwc1m-librealsense-2.28.0 \
  --option binary-caches 'https://cache.nixos.org/ https://r-ryantm.cachix.org/' \
  --option trusted-public-keys '
  r-ryantm.cachix.org-1:gkUbLkouDAyvBdpBX0JOdIiD2/DP1ldF3Z3Y6Gqcc4c=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(r-ryantm's Cachix cache is only trusted for this store-path realization.)

Or, build yourself:
```
nix-build -A librealsense https://github.com/r-ryantm/nixpkgs/archive/3dbfe37976c86b08b4fb6a0bf04f4bc56d2fb604.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/wcqkph9wihh4mg0hkg1vq846iv3rwc1m-librealsense-2.28.0
ls -la /nix/store/wcqkph9wihh4mg0hkg1vq846iv3rwc1m-librealsense-2.28.0/bin
```


</details>
<br/>


cc @brian-dawn for testing.

r-ryantm committed:
librealsense: 2.25.0 -> 2.28.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/librealsense/versions

jonringer commented:
could probably use some multi-output love

```
[nix-shell:~/.cache/nix-review/pr-69657]$ tree ./results/librealsense
./results/librealsense
├── include
│   └── librealsense2
│       ├── h
│       │   ├── rs_advanced_mode_command.h
│       │   ├── rs_config.h
│       │   ├── rs_context.h
│       │   ├── rs_device.h
│       │   ├── rs_frame.h
│       │   ├── rs_internal.h
│       │   ├── rs_option.h
│       │   ├── rs_pipeline.h
│       │   ├── rs_processing.h
│       │   ├── rs_record_playback.h
│       │   ├── rs_sensor.h
│       │   └── rs_types.h
│       ├── hpp
│       │   ├── rs_context.hpp
│       │   ├── rs_device.hpp
│       │   ├── rs_export.hpp
│       │   ├── rs_frame.hpp
│       │   ├── rs_internal.hpp
│       │   ├── rs_options.hpp
│       │   ├── rs_pipeline.hpp
│       │   ├── rs_processing.hpp
│       │   ├── rs_record_playback.hpp
│       │   ├── rs_sensor.hpp
│       │   └── rs_types.hpp
│       ├── rs_advanced_mode.h
│       ├── rs_advanced_mode.hpp
│       ├── rs.h
│       ├── rs.hpp
│       └── rsutil.h
└── lib
    ├── cmake
    │   └── realsense2
    │       ├── realsense2Config.cmake
    │       ├── realsense2ConfigVersion.cmake
    │       ├── realsense2Targets.cmake
    │       └── realsense2Targets-release.cmake
    ├── librealsense2.so -> librealsense2.so.2.28
    ├── librealsense2.so.2.28 -> librealsense2.so.2.28.0
    ├── librealsense2.so.2.28.0
    ├── librealsense-file.a
    └── pkgconfig
        └── realsense2.pc

8 directories, 37 files

```

c0bw3b committed:
librealsense: 2.28.0 -> 2.29.0

+ split libs and includes into multiple outputs
+ use libusb 1.x and not libusb-compat
